src/mlpack/methods/grad_boosting.hpp

```diff
@@ -0,0 +1,17 @@
+/**
+ * @file grad_boosting.hpp
+ * @author Abhimanyu Dayal
+ * 
+ * Convenience include for mlpack/methods/grad_boosting/grad_boosting.hpp.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_GRADBOOSTING_HPP
+#define MLPACK_GRADBOOSTING_HPP
```

Comment:
```suggestion
#ifndef MLPACK_GRAD_BOOSTING_HPP
#define MLPACK_GRAD_BOOSTING_HPP
```
I know, this is insanely pedantic... :)